### ISSUE TITLE
Missing functions and keywords

### ISSUE THREAD

**Issue by MattFryer** [Gimly/vscode-qlik#3]
Hi
I've produced quite a few Qlik language highlightings for Notepad++, UltraEdit, web pages, WordPress, MediaWiki, etc. I was thinking of creating one for VSCode but stumbled across yours. Having tested it looks like a great start! I notice there are quite a lot of missing keywords and functions though. Having done so many language defs I've collated a pretty extensive list of all possible functions and keywords in qlik script. I'd be happy to share and collab on adding the missing ones from this VSCode extension if you'd like?!

Regards
Matt

**Comment by Gimly** [comment]
Hey Matt,

Yes, that would be really nice to get collaboration on this project.

Do you need any help on the implementation side, or on how to collaborate on the project?

Thanks

**Comment by MattFryer** [comment]
Hi Xavier
I've forked the repo and will start taking a look at the grammar definition. From what I've seen so far it's basically just regex to identify the highlighting. 

I've a test script I use when working on any of my syntax highlighting tools. A copy of it can be found [here](https://github.com/MattFryer/Qlik-Web-Highlight/blob/master/development/test-scripts/test%20script.qvs). If you load it up in VSCode it's pretty easy to see which keywords and functions are missing. I've also noticed some keywords are being highlighted when they shouldn't (eg. when they are only valid in combinations). Take a look and see how you want to divide things up. I'm happy to have a go at improving things. 

Regards
Matt
